internal/render/pod.go

```diff
@@ -67,6 +68,7 @@ var defaultPodHeader = model1.Header{
 	model1.HeaderColumn{Name: "%CPU/L", Attrs: model1.Attrs{Align: tview.AlignRight, MX: true}},
 	model1.HeaderColumn{Name: "%MEM/R", Attrs: model1.Attrs{Align: tview.AlignRight, MX: true}},
 	model1.HeaderColumn{Name: "%MEM/L", Attrs: model1.Attrs{Align: tview.AlignRight, MX: true}},
+	model1.HeaderColumn{Name: "GPUS", Attrs: model1.Attrs{Align: tview.AlignRight}},
```

Comment:
I think it might be good to keep consistency with cpu aka `GPU R/L' `%GPU/R' '%GPU/L'  as separate columns one can sort or see where they are at. What do you think?

Reply:
@derailed  Done! I’ve updated it to match the CPU/MEM pattern with the GPU, GPU/RL, %GPU/R, and %GPU/L columns.
As for the container view, I think it’s nice to have, but not a must.
Anyway, I’d like to merge this feature if that’s okay, and start using it :)